handlers/handler-loadbalance/src/test/java/org/apache/servicecomb/loadbalance/TestLoadBalanceHandler2.java

```diff
@@ -335,11 +335,11 @@ public void testZoneAwareAndIsolationFilterWorksEmptyInstanceProtectionEnabled()
     ServiceCombServer server2 = server;
     loadBalancer = handler.getOrCreateLoadBalancer(invocation);
     server = loadBalancer.chooseServer(invocation);
-    Assert.assertEquals(server.getEndpoint().getEndpoint(), "rest://localhost:9090");
+    Assert.assertEquals(server.getEndpoint().getEndpoint(), "rest://localhost:9091");
```

Comment:
1.expected value should be the first parameter
2.why expected value changed?

Reply:
as **IsolationDiscoveryFilter#checkThresholdAllowed** will return false bacause of **serverStats.getFailedRate() >= settings.errorThresholdPercentage.** hence, the instance which matchRegionAndAz is isolated and  it will find instance in other AZ.

Reply:
ok, I will change it.